gridstatus/tests/base_test_iso.py

```diff
@@ -255,11 +255,12 @@ def test_get_load_forecast_today(self):
     """get_status"""
 
     def test_get_status_latest(self):
-        status = self.iso.get_status("latest")
-        assert isinstance(status, GridStatus)
+        if not isinstance(self.iso, gridstatus.spp.SPP):
```

Comment:
rather than skip the test in the base class, we can implement it a test that just checks for not implemented. see example here in pjm tests: https://github.com/kmax12/gridstatus/blob/main/gridstatus/tests/test_pjm.py#L173